core/chains/evm/forwarders/forwarder_manager.go

```diff
@@ -0,0 +1,62 @@
+package forwarders
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/smartcontractkit/chainlink/core/logger"
+	"github.com/smartcontractkit/sqlx"
+)
+
+// Config encompasses config used by fwdmgr
+//go:generate mockery --recursive --name Config --output ./mocks/ --case=underscore --structname Config --filename config.go
+type Config interface {
+	EvmUseForwarders() bool
+	LogSQL() bool
+}
+
+type FwdMgr struct {
+	logger          logger.Logger
+	chainID         big.Int
+	ORM             ORM
+	forwarderMethod abi.Method
+}
+
+var fwdrMethod = makeFwdrMethod()
+
+func NewFwdMgr(db *sqlx.DB, cfg Config, chainId big.Int, lggr logger.Logger) *FwdMgr {
+	lggr = lggr.Named("FwdMgr")
+	lggr.Infow("Initializing EVM forwarder manager")
+
+	fwdMgr := FwdMgr{
+		logger:  lggr,
+		chainID: chainId,
+		ORM:     NewORM(db, lggr, cfg),
+	}
+	return &fwdMgr
+}
+
+func (fm *FwdMgr) MakeForwardedPayload(to common.Address, EncodedPayload []byte) (fwdPayload []byte) {
+	fwdPayload = append(fwdrMethod.ID, to.Bytes()...)
+	fwdPayload = append(fwdPayload, EncodedPayload...)
```

Comment:
```suggestion
	fwdPayload = append(fwdrMethod.ID, to.Bytes()...)
	fwdPayload = append(common.BigToHash(64).Bytes()) // location of the dynamic length variable
	fwdPayload = append(common.BigToHash(EncodedPayload.length).Bytes()) // length of second arg
	fwdPayload = append(fwdPayload, EncodedPayload...)
```
Have you tested this method with an actual contract? I believe [Ethereum's ABI is a bit more complicated](https://docs.soliditylang.org/en/develop/abi-spec.html#use-of-dynamic-types), the above is psuedo code because we may just want to hardcode the location variable somewhere else.

We should be able to use the `abi` package to do a lot of this formatting for us so that we don't have to know the ABI specifics. I believe @RyanRHall would be able to point you to some examples of that.